plugins/callback/counter_enabled.py

```diff
@@ -143,13 +137,13 @@ def v2_playbook_on_task_start(self, task, is_conditional):
         # that they can secure this if they feel that their stdout is insecure
         # (shoulder surfing, logging stdout straight to a file, etc).
         if not task.no_log and C.DISPLAY_ARGS_TO_STDOUT:
-            args = ', '.join(('%s=%s' % a for a in task.args.items()))
-            args = ' %s' % args
-        self._display.banner("TASK %d/%d [%s%s]" % (self._task_counter, self._task_total, task.get_name().strip(), args))
+            args = ', '.join(('{k}={v}' for k, v in task.args.items()))
+            args = f' {args}'
+        self._display.banner(f"TASK {int(self._task_counter)}/{int(self._task_total)} [{task.get_name().strip()}{args}]")
```

Comment:
`int()` isn't needed, it's automatically formatted as an integer:
```suggestion
        self._display.banner(f"TASK {self._task_counter}/{self._task_total} [{task.get_name().strip()}{args}]")
```
Same below.

Reply:
Yeah, I thought so. I have used this tool `flynt` to bulk-convert the templating and it ~~chickens out~~ refuses to transform interpolations of `%d` in its normal execution with:
```
Not converting code due to: Skipping %d formatting - fstrings behave differently from % formatting.
```
Then I used a `-a` (for "aggressive") transformation, and that's what it did.